examples/chart/teleport-cluster/.lint/proxy-hpa.yaml

```diff
@@ -0,0 +1,14 @@
+clusterName: test-cluster
+chartMode: aws
+aws:
+  region: us-west-2
+  backendTable: test-dynamodb-backend-table
+  auditLogTable: test-dynamodb-auditlog-table
+  sessionRecordingBucket: test-s3-session-storage-bucket
+proxy:
+  teleportConfig: {}
```

Comment:
```suggestion
```

I don't think this value is required when nothing is being overridden.

Reply:
I think we can remove `teleportConfig: {}` as its not required